### PR TITLE
Add a message when duplicating Variables or GI upon importation of Ta…

### PR DESCRIPTION
**Pull request by Aminelouati**
Add a message when duplicating Variables or GI upon importation of Tasks in Studio.

### CODE REVIEW

File: app/scripts/proactive/model/Job.js [hunk 1]
```diff
@@ -508,22 +511,55 @@ define(
...
         console.log("Fetching documentation in GI...");
 
         // Get the job name and put it as name for the generated file
-        var genericInformation = this.attributes["Generic Info"];
+
         var fileContent = "";
         var linkName = "Undefined";
         var documentationValue = "Undefined";
 
+        console.log('genericInformation',genericInformation);
```
Comment (zeineb) [inline app/scripts/proactive/model/Job.js]: Maybe you need to delete this line.

File: app/scripts/proactive/model/Job.js [hunk 2]
```diff
@@ -508,22 +511,55 @@ define(
...
         return ["Name", "Variables"]
       },
-      generateDocumentUrl: function() {
+
+      findDuplicates: function(type, inputArray, property, value, myStack){
+          console.log("Identification of duplicated " + type + "...");
+          var notDuplicatedElements = new Set();
+          var indexOfDuplicatedElements = new Set();
+          var  j = 0;
+          var list = inputArray.map(function(item){ return item[property] });
```
Comment (zeineb) [inline app/scripts/proactive/model/Job.js]: We need maybe to test if "list" is not null?
